corehq/apps/users/models.py

```diff
@@ -1088,13 +1088,19 @@ def sync_to_django_user(self):
             django_user = User(username=self.username)
         for attr in DjangoUserMixin.ATTRS:
             attr_val = getattr(self, attr)
-            if not attr_val and attr != 'last_login':
+            if attr in [
+                'is_active',
+                'is_staff',
+                'is_superuser',
+            ]:
+                attr_val = bool(attr_val)
```

Comment:
is this just to set a default when `attr_val` is None? if so it might be better to just explicitly default it to False when it's None.